### PR TITLE
docs: prebuilt executor

### PR DESCRIPTION
**Pull request by tkonieczny**

### CODE REVIEW

File: docs/docs/test-types/legacy-executor.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
-# Custom Testkube Executor
+# Legacy Testkube Executor
```
Comment (vsukhin) [inline docs/docs/test-types/legacy-executor.md:1]: Prebuilt

**Review by vsukhin — COMMENTED** [review]
one comment

File: docs/docs/test-types/legacy-executor.md (hunk 1 above)
Comment (tkonieczny) [reply docs/docs/test-types/legacy-executor.md]: updated

**Review by vsukhin — APPROVED** [review]

File: docs/docs/test-types/container-executor.mdx [hunk 2]
```diff
@@ -7,6 +7,8 @@ The Testkube Container Executor allows you to run your own container images for
 
 The Test execution fails if the container exits with an error and succeeds when the container command successfully executes.
 
+That's the new, and recommended way of running custom images. Docs about the prebuilt custom executors can be found [here](prebuilt-executor.md).
```
Comment (jfermi) [inline docs/docs/test-types/container-executor.mdx:10]:
```suggestion
That's the new and recommended way of running custom images. Docs about the prebuilt custom executors can be found [here](prebuilt-executor.md).
```

File: docs/docs/test-types/prebuilt-executor.md [hunk 3]
```diff
@@ -1,4 +1,7 @@
-# Custom Testkube Executor
+# Prebuilt Testkube Executor
+
+**The new, and recommended way of running custom images is to use [Container Executors](container-executor.mdx).**
```
Comment (jfermi) [inline docs/docs/test-types/prebuilt-executor.md:3]:
```suggestion
**The new and recommended way of running custom images is to use [Container Executors](container-executor.mdx).**
```

**Review by jfermi — COMMENTED** [review]
A couple of editing suggestions.

**Pull request merged**